benchmarks/src/util/run.rs

```diff
@@ -138,6 +144,28 @@ impl BenchmarkRun {
         }
     }
 
+    /// Print the names of failed queries, if any
+    pub fn maybe_print_failures(&self) {
+        let failed_queries: Vec<&str> = self
+            .queries
+            .iter()
+            .filter_map(|q| (!q.success).then_some(q.query.as_str()))
+            .collect();
+
+        if !failed_queries.is_empty() {
+            println!("Failed Queries: {}", failed_queries.join(", "));
+        }
+    }
```

Comment:
Since we call `q.query.as_str` here, expected output may varies in benchmarks. 
For example, 
```
// In sort_tpch
Failed Queries: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11
// In clickbench
Failed Queries: Query 4, Query 5, Query 7
```